### PR TITLE
Working deploy

### PR DESCRIPTION
**Pull request by andrewhojel**
added a firebase.json with proper routing and changes the name of the login and signup pages to ensure no confusing when routing paths (it now deploys!!)

### CODE REVIEW

**Comment by angelinechen9** [comment]
I fixed the following issues:

- I got the default message “Firebase Hosting Setup Complete” instead of the home page.  I learned that I need to run npm run build and change the public directory from build to view/build in firebase.json in order to upload the folder that contains the Hosting assets.
- I got the error message “Page Not Found” instead of the sign up page.  I learned that I need to configure the web application as a single-page app in order to rewrite all urls to /index.html.

**Comment by angelinechen9** [comment]
I fixed the following issues:

- I got the default message “Firebase Hosting Setup Complete” instead of the home page.  I learned that I need to run npm run build and change the public directory from build to view/build in firebase.json in order to upload the folder that contains the Hosting assets.
- I got the error message “Page Not Found” instead of the sign up page.  I learned that I need to configure the web application as a single-page app in order to rewrite React Router URLs to /index.html.
- I got a HTML file instead of the response returned by the server.  I learned that I need to rewrite API URLs to the Cloud Function.

File: README.md [hunk 1]
```diff
@@ -129,6 +129,12 @@ Here is a very basic description of the repository structure:
     npm run build
     firebase deploy
 
+## Deployment Notes
+
+- Run `npm run build` and change the public directory from build to view/build in firebase.json in order to upload the folder that contains the Hosting assets.
+- Configure the web application as a single-page app in order to rewrite React Router URLs to /index.html.
+- Rewrite API URLs to the Cloud Function.
```
Comment (jessimb) [inline README.md:136]: How does one do these things?

**Review by jessimb — APPROVED** [review]
Little more detail in the readme, but good to go after that

File: README.md (hunk 1 above)
Comment (angelinechen9) [reply README.md:136]: The [documentation](https://firebase.google.com/docs/hosting/full-config) describes how to rewrite URLs.

**Pull request merged**